### DISCUSSION TITLE
SSH Client Java 8

### DISCUSSION THREAD

**Discussion by franciscoamores** [hierynomus/sshj#765] [Q&A] [answered]
Hello,

What version of SSHJ should work fine with Java8?

We get NoClassDefFoundError: java.lang.NoClassDefFoundError: Could not initialize class net.schmizz.sshj.common.KeyType

Is there any reference for java version compatibility?

Thanks!

**Comment by hierynomus** [comment, marked as answer]
It seems that you've then got half a jar somehow, as `KeyType` should be part of the SSHJ jar itself.
